closure/goog/array/array.js

```diff
@@ -1417,7 +1435,8 @@ goog.array.bucket = function(array, sorter, opt_obj) {
 goog.array.toObject = function(arr, keyFunc, opt_obj) {
   var ret = {};
   goog.array.forEach(arr, function(element, index) {
-    ret[keyFunc.call(/** @type {?} */ (opt_obj), element, index, arr)] = element;
+    ret[keyFunc.call(/** @type {?} */ (opt_obj), element, index, arr)] =
+        element;
```

Comment:
This seems unrelated, though it does look like we had an issue with >80 columns.  Might be more readable to just pull out

    var key = keyFunc.call(...);

rather than wrapping the line.